base/replutil.jl

```diff
@@ -93,6 +93,13 @@ function showerror(io::IO, ex::LoadError, bt; backtrace=true)
 end
 showerror(io::IO, ex::LoadError) = showerror(io, ex, [])
 
+function showerror(io::IO, ex::InitError, bt; backtrace=true)
+    print(io, "InitError: ")
+    showerror(io, ex.error, bt, backtrace=backtrace)
+    print(io, "\nduring initialization of module $(ex.mod)")
```

Comment:
@jakebolewski, It seems like @phobon's code is correct; it should be analogous to `showerror(io::IO, ex::LoadError)`, and you need to show `ex.error` at minimum.